cockatrice/src/window_main.cpp

```diff
@@ -375,10 +378,12 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
             }
             break;
         case Response::RespClientIdRequired:
-            QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\nPlease close and reopen your client to try again."));
+            QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\n"
```

Comment:
Why are you splitting these? It helps translators when it's one thing in context. They should also control the spacing and line breaks since the lines can be very different lengths from english.